java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/ConnectionI.java

```diff
@@ -20,8 +20,9 @@
 import java.util.concurrent.TimeUnit;
 import java.util.function.Consumer;
 
+// Public because it's used by the 'Ice/metrics' test.
 /**
- * @hidden Public because it's used by the 'Ice/metrics' test.
+ * @hidden
  */
```

Comment:
For what it's worth, there's lots of these kinds of `@hidden` tags in our source code.